### ISSUE TITLE
Unexpected Identifier

### ISSUE THREAD

**Issue by ddaaggeett** [itinance/react-native-fs#285]
after following the setup instructions, i keep getting this upon running the react-native server:

    /Users/.../node_modules/react-native-fs/FS.common.js:29
    type MkdirOptions = {
         ^^^^^^^^^^^^
    SyntaxError: Unexpected identifier
            at createScript (vm.js:53:10)
            ...

**Comment by itinance** [comment]
Any news on this? I did several installations those days on newly created RN-projects with each PR that was sent and got never this error

**Comment by ddaaggeett** [comment]
Yea, there's another issue I opened in another repo, [here](https://github.com/wkh237/react-native-fetch-blob/issues/344), and I believe it also has something to do with `babel` but haven't had much time to look more into it the past couple days.

I created the RN app the recommended/default way: `react-native init <someProject>` and by the looks of it, `es2015` should automatically be running in the latest react-native.

I have my server start with the import:

`var rnfs = require('react-native-fs');` which results:

    /Users/.../node_modules/react-native-fs/FS.common.js:27
    var normalizeFilePath = (path: string) => (path.startsWith('file://') ? path.slice(7) : path);
                                 ^
    SyntaxError: Unexpected token :
        at createScript (vm.js:53:10)

I'm not quite sure how I was getting the error I posted initially, but this is the error I'm getting now.

If it helps, I'm using `node.js v7.7.2` and `npm v4.1.2`
